backend/src/scripts/migrations/4.30/addVertexCountsToMeshBSONs.js

```diff
@@ -34,22 +34,26 @@ const processModel = async (teamspace, model) => {
 	const superMeshes = await find(teamspace, stashDB, { type: 'mesh' }, { m_map: 1 });
 
 	const spanningMeshVerticesCounts = [];
-	const meshUpdates = [];
 
 	// eslint-disable-next-line camelcase
-	superMeshes.forEach(({ m_map }) => {
+	superMeshes.forEach(async ({ m_map }) => {
```

Comment:
if you have to do async use map to capture all the promises, as it stands there's no guarantee your promises are done before you exit the function